shared/ui-components/src/forms/Select.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { ErrorMessage } from "./ErrorMessage"
+import { FormOptions } from "../helpers/formOptions"
+
+export const Select = ({
+  error,
+  errorMessage,
+  controlClassName,
+  id,
+  name,
+  label,
+  defaultValue,
+  placeholder,
+  register,
+  validation,
+  disabled,
+  options,
+  keyPrefix,
+}: Props) => {
+  return (
+    <div className={"field " + (error ? "error" : "")}>
+      <label className="field-label--caps" htmlFor={id}>
+        {label}
+      </label>
+      <div className={controlClassName}>
+        <select
+          className="input"
+          id={id || name}
+          name={name}
+          defaultValue={defaultValue}
+          ref={register(validation)}
+          disabled={disabled}
+        >
+          {placeholder && (
+            <option value="" disabled selected>
+              {placeholder}
+            </option>
+          )}
+          <FormOptions options={options} keyPrefix={keyPrefix} />
+        </select>
+      </div>
+      {error && errorMessage && <ErrorMessage error={error}>{errorMessage}</ErrorMessage>}
+    </div>
+  )
+}
+
+interface Props {
```

Comment:
For this and the checkbox group, it'd be great to add a component name prefix to the interface (aka `SelectProps`)